.github/workflows/test-e2e.yml

```diff
@@ -17,6 +17,7 @@ jobs:
         testcase:
           - TestE2ESimple
           - TestCelestiaChainStateSync
+          - TestE2ENodeSimple
```

Comment:
[uber nit][optional] `Simple` makes me conflate this with `TestE2ESimple`. To disambiguate, consider renaming to `TestE2EBridgeNode`

Reply:
yes good call, will make this change.